polkadot-host-spec/c02-state.tm

```diff
@@ -16,6 +16,14 @@
   limits on their length. The limit is imposed by the encoding algorithms to
   store the key and the value in the storage trie.
 
+  \;
+
+  The state storage requires a mechanism to identify any changes made by each
+  block, in case non-finalized blocks have to be reverted or a Changes Trie
+  as described in section <reference|sect-general-tree-structure> needs to be
```

Comment:
Could you clarify  the meaning of "identify any changes made by each block"? Does that mean that there should be a mechanism to get the list of keys changed by a block for any block in the block tree?